### DISCUSSION TITLE
How can I inject resolver implementation through plugin ?

### DISCUSSION THREAD

**Discussion by wenerme** [99designs/gqlgen#1805] [Q&A]
### What happened?
Look at resolvergen, try to get a way to add generated resolver method body, but found the code use internal rewrite to get the imp from code, it's not possible from plugin

### What did you expect?

Have to hook to inject resolver for type/field

**Comment by cliedeman** [comment]
@wenerme i started work on a plug-in for ent here. 

https://github.com/ent/contrib/pull/78

I also have investigated how to generated resolvers

**Comment by wenerme** [comment]
@cliedeman thanks, currently I using a rewriter to do this, works fine, change gqlgen allowed inject is better.
